content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/session-certificates.md

```diff
@@ -0,0 +1,77 @@
+---
+date: '2022-11-15'
+title: "Configuring Optional Session Certificates"
+menu:
+  corda-5-alpha:
+    identifier: corda-5-alpha-mgm-onboarding
```

Comment:
I'm not too familiar with these headers but these seems like it's referencing the wrong page

Reply:
> I'm not too familiar with these headers but these seems like it's referencing the wrong page

good spot, thanks!